src/ils.cpp

```diff
@@ -100,56 +100,57 @@ usage() {
     char *msgs[] = {
         "Usage: ils [-ArlLv] dataObj|collection ... ",
         "Usage: ils --bundle [-r] dataObj|collection ... ",
-        "Display data Objects and collections stored in irods.",
+        "Display data objects and collections stored in iRODS.",
         " ",
         "The following is typical output for ils with no arguments:"
         "    $ ils",
         "    /tempZone/public:",
         "      big",
         "      foo",
         "The only information displayed are the names of data objects in the current",
-        "working directory.",
+        "working collection.",
         " ",
         "The long format (given by -l) will display some more information:",
         "    $ ils -l",
         "    /tempZone/public:",
-        "      rods              0 demoResc     41943040 2021-04-29.17:57 & big",
-        "      rods              2 comp;arch    40236581 2021-04-19.03:00 X big",
-        "      alice             2 repl1;child1          283 2021-04-29.17:54 ? foo",
-        "      alice             3 repl1;child2          283 2021-04-29.17:56 ? foo",
-        "      alice             4 repl1;child3          283 2021-04-29.17:57 ? foo",
+        "      rods              0 demoResc              41943040 2021-04-29.17:57 & big",
```

Comment:
gah, this is not better.    i see now with local testing that they don't actually all line up like this together... they're kind of all over the place.

if we're going to manipulate this... can we right justify the file size column?  i think that's what will be most readable.

Reply:
very good - yes please.